### PR TITLE
[Overflow] Add refresh method, add VisibleOnLoad parameter

### PR DESCRIPTION
**Pull request by vnbaaij**
- Rename functions in js to be in line with rest of scripts
- Add public RefreshAsync method to call refresh function in script (#2225)
- Add VisibleOnLoad parameter (#2224)

### CODE REVIEW

File: src/Core/Components/Overflow/FluentOverflow.razor.cs [hunk 1]
```diff
@@ -88,8 +90,16 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
         {
             _jsModule = await JSRuntime.InvokeAsync<IJSObjectReference>("import", JAVASCRIPT_FILE);
             _dotNetHelper = DotNetObjectReference.Create(this);
+            await _jsModule.InvokeVoidAsync("fluentOverflowInitialize", _dotNetHelper, Id, IsHorizontal, Selectors);
```
Comment (dvoituron) [inline src/Core/Components/Overflow/FluentOverflow.razor.cs:93]: Could you call `RefreshAsync` to have only one call to this JS method?

**Review by dvoituron — CHANGES_REQUESTED** [review]

File: src/Core/Components/Overflow/FluentOverflow.razor.cs (hunk 1 above)
Comment (vnbaaij) [reply src/Core/Components/Overflow/FluentOverflow.razor.cs:93]: No, the 'initialize code' is different JS call then for the 'refresh code'. Initialize sets up the MutationObserver and ResizeObserver and then calls the (re)calculation on the JS side. Resize just calls the (re)calculation

Comment (dvoituron) [reply src/Core/Components/Overflow/FluentOverflow.razor.cs:93]: Hups... I didn't have my glasses ;-)

File: src/Core/Components/Overflow/FluentOverflow.razor.cs [hunk 2]
```diff
@@ -24,6 +25,7 @@ public partial class FluentOverflow : FluentComponentBase, IAsyncDisposable
 
     /// <summary />
     protected string? StyleValue => new StyleBuilder(Style)
+        .AddStyle("visibility",_visibility)
```
Comment (dvoituron) [inline src/Core/Components/Overflow/FluentOverflow.razor.cs:28]: Probably better to use `.AddStyle("visibility", "hidden", when: VisibleOnLoad == true)`.
So, you can remove the `_visibility` variable and the style will be applied only when the property is True (and the user could override the style in other cases).

**Review by dvoituron — CHANGES_REQUESTED** [review]

**Review by dvoituron — APPROVED** [review]

**Pull request merged**